source/sdk/lang/types.ooc

```diff
@@ -84,12 +85,30 @@ Bool: cover from bool {
 Closure: cover {
 	thunk: Pointer
 	context: Pointer
-	free: func {
-		if (this context != null) {
+	_owner: Owner
+	owner ::= this _owner
+	isOwned ::= this _owner != Owner Unknown && this _owner != Owner Static && this _owner != Owner Stack && this context != null
```

Comment:
Not in this PR, but this code `this _owner != Owner Unknown && this _owner != Owner Static && this _owner != Owner Stack` is now repeated in few files. Maybe we should move it to `OwnedBuffer` or `Owner`.